src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+    private static final String DELIMITERS = "\\s*[^a-zA-Z]+\\s*";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder information = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                information.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("File can't be read ", e);
+        }
+        String[] words = information.toString().toLowerCase().split(DELIMITERS);
+        String[] wordsStartsOnSpecifiedCharacter =
+                new String[getCountWordsStartsOnSpecifiedCharacter(words)];
+        int index = 0;
+        for (String word : words) {
+            if (startWithLetter(word)) {
+                wordsStartsOnSpecifiedCharacter[index] = word;
+                index++;
+            }
+        }
```

Comment:
you iterating twice through all words, first time to get the number, and second while initialization. I know that you want to have the array size before initializing each element, but try to do this in the range of one iteration

Reply:
done